third_party/libusb/webport/src/libusb-to-webusb-adaptor.js

```diff
@@ -342,6 +330,40 @@ GSC.LibusbToWebusbAdaptor = class extends GSC.LibusbToJsApiAdaptor {
     return libusbJsDevice;
   }
 
+  /**
+   * Fills the extraData fields of the passed configurations, by fetching and
```

Comment:
nit: "...of all the passed configurations..."

This is just a personal preference to avoid confusion with the comment for the function handling a single passed configuration

Reply:
"of all passed configurations" - also, don't follow that by a ","